R/ets.R

```diff
@@ -206,21 +206,21 @@ specials_ets <- new_specials(
 #' }
 #'
 #' \tabular{ll}{
-#'   `method`     \tab The form of the error term: either additive ("A") or multiplicative ("M"). If the error is multiplicative, the data must be non-negative.
+#'   `method`     \tab The form of the error term: either additive ("A") or multiplicative ("M"). If the error is multiplicative, the data must be non-negative. The default is to choose this from the data.
```

Comment:
I think rather than describing the default, this could be generalised to say what happens when multiple `method` values are specified.

That is to say all specified `method` options will be tested on the data and the model that gives the lowest `ic` will be kept.